target/supervisor/conf.d/supervisor-app.conf

```
@@ -157,3 +157,15 @@ autostart=false
 stdout_logfile=/var/log/supervisor/%(program_name)s.log
 stderr_logfile=/var/log/supervisor/%(program_name)s.log
 command=/bin/bash -l -c /usr/local/bin/update-check.sh
+
+# Docs: https://github.com/Snawoot/postfix-mta-sts-resolver/blob/master/man/mta-sts-daemon.1.adoc
+[program:mta-sts-daemon]
+startsecs=0
+stopwaitsecs=55
+autostart=false
+autorestart=true
+stdout_logfile=/var/log/supervisor/%(program_name)s.log
+stderr_logfile=/var/log/supervisor/%(program_name)s.log
+command=/usr/bin/python3 /usr/bin/mta-sts-daemon --config /etc/mta-sts-daemon.yml
```

This will work in the meantime, since the script called is like the fail2ban python script which has a python3 shebang.

The shebang is sufficient for the command to be run without directly running the script via the python3 command. That affects the pgrep matching without `--full`.

```suggestion
command=/usr/bin/mta-sts-daemon --config /etc/mta-sts-daemon.yml
```